### PR TITLE
Tech spa

### PR DESCRIPTION
**Pull request by AaronGullickson**
Some simple SPAs for techs that allow them to specialize (a -1 bonus) when working with particular part types. I also wanted to add a non-weapon equipment specialization but that is not specifically tracked with the repair part types currently.

### CODE REVIEW

File: MekHQ/docs/history.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ v0.45.2-SNAPSHOT
 + Switched build system from ant to gradle.
 + Issue #917: Hire Full Compliment Regularly Hires Bondsmen
 + Issue #919: Dropship Crewmembers/techs displayed as "Nobody" In Repair Bay
++ added tech specialist SPAs
```
Comment (sixlettervariables) [inline MekHQ/docs/history.txt]: We'll modify history.txt after the change lands, still you can revert this change.

**Comment by AaronGullickson** [comment]
Ahh, I was wondering how you were handling the history log with the pull request system. I have removed that entry.
